django/core/management/commands/createcachetable.py

```diff
@@ -53,55 +54,32 @@ def create_table(self, database, tablename, dry_run):
                 self.stdout.write("Cache table '%s' already exists." % tablename)
             return
 
-        fields = (
+        class CacheTable(models.Model):
             # "key" is a reserved word in MySQL, so use "cache_key" instead.
-            models.CharField(name='cache_key', max_length=255, unique=True, primary_key=True),
-            models.TextField(name='value'),
-            models.DateTimeField(name='expires', db_index=True),
-        )
-        table_output = []
-        index_output = []
-        qn = connection.ops.quote_name
-        for f in fields:
-            field_output = [
-                qn(f.name),
-                f.db_type(connection=connection),
-                '%sNULL' % ('NOT ' if not f.null else ''),
-            ]
-            if f.primary_key:
-                field_output.append("PRIMARY KEY")
-            elif f.unique:
-                field_output.append("UNIQUE")
-            if f.db_index:
-                unique = "UNIQUE " if f.unique else ""
-                index_output.append(
-                    "CREATE %sINDEX %s ON %s (%s);" %
-                    (unique, qn('%s_%s' % (tablename, f.name)), qn(tablename), qn(f.name))
-                )
-            table_output.append(" ".join(field_output))
-        full_statement = ["CREATE TABLE %s (" % qn(tablename)]
-        for i, line in enumerate(table_output):
-            full_statement.append('    %s%s' % (line, ',' if i < len(table_output) - 1 else ''))
-        full_statement.append(');')
+            cache_key = models.CharField(max_length=255, unique=True, primary_key=True)
+            value = models.TextField()
+            expires = models.DateTimeField(db_index=True)
 
-        full_statement = "\n".join(full_statement)
+            class Meta:
+                apps = Apps()
+                app_label = '<ignored due to Apps()>'
```

Comment:
Could use `cache` here to be consistent with how the migration recorder does it.
```suggestion
                app_label = 'cache'
```